10/umbraco-cms/fundamentals/design/rendering-media.md

```diff
@@ -78,6 +78,40 @@ As with example one, we are accessing a MediaType `image` using the same Guid as
 It is always worth having null-checks around your code when retrieving media in case the conversion fails or Media() returns null. This makes your code more robust.
 {% endhint %}
 
+## Working with Video files
+
+If you upload a video file (such as .mp4) to the Media library, Umbraco will store it as a Video Media Type by default. Unlike images, video files won’t include properties like `umbracoWidth` or `umbracoHeight`, but you can still retrieve the media item and render it using the `<video>` HTML tag.
+
+### Example: Rendering a Video Media item
+
+```csharp
+@{
+    var videoUrl = string.Empty;
+    var videoMediaItem = Umbraco.Media(Guid.Parse("8c49c5d3-cb2a-48b2-87c9-7e2c1873e948"));
+
+    if (videoMediaItem != null)
+    {
+        videoUrl = videoMediaItem?.Url();
+    }
+}
+      
+@if (!string.IsNullOrEmpty(videoUrl))
+{
+    <video width="640" height="360" controls>
+        <source src="@videoUrl" type="video/mp4">
+            Your browser does not support the video tag.
+    </video>
+}
+```
+
+This example assumes that:
+
+* You've uploaded an MP4 file to the **Media** section.
```

Comment:
Earlier in the article you refer to this file type using .mp4 - does it make sense to chose one of the two? For consistency, I mean 🤔